Scripts/accountDataVerification.js

```diff
@@ -0,0 +1,25 @@
+//메일 검증용 API링크 https://bootcamp-api.codeit.kr/docs
+
+const emailRegEx = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+const pwRegEx = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
+
+
+const EmailRegisterStatusInServer = async function (inputtedAccountEmail) {
```

Comment:
대문자로 시작하는 것보다, pascal case보다 camel case로 하시는게 더 일반적일것 같아요.
```suggestion
const registerEmailStatusInServer = async function (inputtedAccountEmail) {
```

Reply:
Pascal이 손에 익어서 이런 잔실수가 나오네요... 확인했습니다!